unfetter-discover/unfetter-ui
# Issue 1133/obj with baselines

Requires unfetter-discover/unfetter#1086
(PR: https://github.com/unfetter-discover/unfetter-ui/pull/506)
Requires unfetter-discover/unfetter#1171
(PR:  https://github.com/unfetter-discover/unfetter-store/pull/207)
Fixes unfetter-discover/unfetter#1133

As capabilities are selected for chosen capability groups, an object assessment is created and stored to the DB, and the ID for that object assessment is added to the current baseline (i.e. assessment set).

### To test:
1. Pull this branch
2. Create a new baseline and, before selecting any capabilities, verify assessment set exists in DB without any object assessments in the "assessments" array
3. Select a capability in the baselines wizard, and verify in the DB that a new object assessment has been created and added to the assessment set:
db.getCollection('stix').find({"stix.type" : "x-unfetter-assessment-set"})
db.getCollection('stix').find({"stix.type" : "x-unfetter-object-assessment"})
